core/src/apps/benfen/sign_message.py

```
@@ -0,0 +1,47 @@
+    bfc_address = try_convert_to_bfc_address(address)
+    if bfc_address is None:
+        raise wire.DataError("bfc_address is none")
+
+    len_bytes = uleb_encode(len(msg.message))
+    intentMessage = PERSONALMESSAGE_INTENT_BYTES + len_bytes + msg.message
+
+    from trezor.ui.layouts import confirm_signverify
+
+    ctx.primary_color, ctx.icon_path = lv.color_hex(PRIMARY_COLOR), ICON
+    await confirm_signverify(
+        ctx, "Benfen", decode_message(msg.message), bfc_address, False
```

Ruff (0.8.2)

Boolean positional value in function call

(FBT003)